kms/attestations/verify_attestation_test.py

```diff
@@ -0,0 +1,133 @@
+# Copyright 2020 Google, LLC.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import base64
+import tempfile
+import verify_attestation
```

Comment:
nit: imports should be 1) std libs 2) third party libs 3) package local libs, separated with an empty linne.

```python
import base64
import tempfile

import verify_attestation
```